### ISSUE TITLE
KeyError: 'stateProv'

### ISSUE THREAD

**Issue by Ostyk** [tomas-net/ip2geotools#6]
When applying the following function to a quite large (2k) pandas column of IP addresses
```python
from ip2geotools.databases.noncommercial import DbIpCity
def ip_location(ip_address):
    response = DbIpCity.get(ip_address) 
    return  (response.country, response.city)
```
roughly after processing 10% of it, the system return this error

```python
KeyError                                  Traceback (most recent call last)
<ipython-input-5-f4580d09d702> in <module>
      1 tqdm.pandas(desc="IP!")
----> 2 df['User_location'] = df['User IP'].progress_apply(utils.ip_location)

/usr/local/lib/python3.6/dist-packages/tqdm/_tqdm.py in inner(df, func, *args, **kwargs)
    720                 # Apply the provided function (in **kwargs)
    721                 # on the df using our wrapper (which provides bar updating)
--> 722                 result = getattr(df, df_function)(wrapper, **kwargs)
    723 
    724                 # Close bar and return pandas calculation result

/usr/local/lib/python3.6/dist-packages/pandas/core/series.py in apply(self, func, convert_dtype, args, **kwds)
   4040             else:
   4041                 values = self.astype(object).values
-> 4042                 mapped = lib.map_infer(values, f, convert=convert_dtype)
   4043 
   4044         if len(mapped) and isinstance(mapped[0], Series):

pandas/_libs/lib.pyx in pandas._libs.lib.map_infer()

/usr/local/lib/python3.6/dist-packages/tqdm/_tqdm.py in wrapper(*args, **kwargs)
    716                     # take a fast or slow code path; so stop when t.total==t.n
    717                     t.update(n=1 if not t.total or t.n < t.total else 0)
--> 718                     return func(*args, **kwargs)
    719 
    720                 # Apply the provided function (in **kwargs)

/mnt/c/Users/michal/Desktop/MateCat-timings/code/utils.py in ip_location(ip_address)
     79 
     80 def ip_location(ip_address):
---> 81     response = DbIpCity.get(ip_address)
     82     return  (response.country, response.city)

/usr/local/lib/python3.6/dist-packages/ip2geotools/databases/noncommercial.py in get(ip_address, api_key, db_path, username, password)
     65         # format data
     66         ip_location.country = content['countryCode']
---> 67         ip_location.region = content['stateProv']
     68         ip_location.city = content['city']
     69 

KeyError: 'stateProv'
```

**Comment by tomas-net** [comment]
Which IP address are you trying to locate when `KeyError: 'stateProv'` occurs? It looks like stateProv is not provided by geolocation database.

**Comment by bgehman** [comment]
trivially reproducible when trying to lookup an address in the private range:
```
192.168.0.0 - 192.168.255.255
172.16.0.0 - 172.31.255.255
10.0.0.0 - 10.255.255.255
```
or loop back range:
```
127.0.0.0 – 127.255.255.255
```

Could have a better error instead of just `KeyError`...

**Comment by camontanezp** [comment]
I got the same error for this IP: 49.245.46.219

**Comment by guyts** [comment]
Was this issue ever resolved? Getting similar error here

**Comment by peeves91** [comment]
got the same error here, just installed the package as the latest version, ip that fails is 45.60.16.237

**Comment by premma** [comment]
@tomas-net This bug could be easily fixed by adding 
```
     65         # format data
     66         ip_location.country = content['countryCode']
---> 67         ip_location.region = content.get('stateProv', '')
     68         ip_location.city = content['city']
     69 
```
Plus accordingly to do it at all fields that fail. 

Is it possible to make this fix after almost two years? :-)

**Comment by Gunjan21** [comment]
> @tomas-net This bug could be easily fixed by adding
> 
> ```
>      65         # format data
>      66         ip_location.country = content['countryCode']
> ---> 67         ip_location.region = content.get('stateProv', '')
>      68         ip_location.city = content['city']
>      69 
> ```
> 
> Plus accordingly to do it at all fields that fail.
> 
> Is it possible to make this fix after almost two years? :-)

This should actually be done for the country and city as well.

**Comment by tomas-net** [comment]
Should be fixed in 0.1.6 now.
